machines/bleShare/scan/scanMachine.ts

```diff
@@ -508,7 +514,10 @@ export const scanMachine =
                 },
                 ACCEPT_REQUEST: {
                   target: 'sendingVc',
-                  actions: 'setShareLogTypeUnverified',
+                  actions: [
+                    'setShareLogTypeUnverified',
+                    'updateFaceCaptureBannerStatus',
```

Comment:
can we use `resetFaceCaptureBannerStatus` action here directly?

Reply:
Addressed